src/utils.ts

```diff
@@ -639,14 +685,44 @@ export async function getEnsOwner(
     );
     owner = await ensNameWrapperContract.ownerOf(ensHash);
   }
-  return owner;
+
+  if (owner === EMPTY_ADDRESS && domainType === 'other-tld') {
+    const resolvedAddress = await provider.resolveName(ens);
+
+    // Filter out domains with valid TXT records, but not imported
+    if (resolvedAddress) {
+      owner = await getDNSOwner(ens);
+    }
+  }
+
+  if (owner === EMPTY_ADDRESS && domainType === 'subdomain') {
+    try {
+      owner = await provider.resolveName(ens);
+    } catch (e: any) {
+      // mute error from coinbase, when the subdomain is not found
+      // most other resolvers just return an empty address
+      const coinbaseError = e.message.includes(
+        'response not found during CCIP fetch'
+      );
+      // mute error from missing offchain resolver on sepolia
+      const sepoliaMissingResolverError =
+        e.message.includes('UNSUPPORTED_OPERATION') && network === '11155111';
```

Comment:
Is there some other way than hardcoding sepolia? this testnet can change time to time

Reply:
Maybe we can just ignore all unsupported operations irrespective of the network?